### ISSUE TITLE
Automatically repeat all characters when entering

### ISSUE THREAD

**Issue by leezhian** [BloopAI/bloop#1234]
**Describe the bug**
Automatically repeat the entire text when entering Chinese.

**Expected behavior**
Using Chinese input normally in conversations.

**To Reproduce**
How can we reproduce the bug? Helpful information could include:
device：macbook m1 pro
system：macos 14.2.1
input software：xunfei v1.1.1004 (https://srf.xunfei.cn/#/)
lang：Chinese

input: nihao
expected output：你好
error output：nnnninini'hni'hani'hani'hao你好

**Screenshots or output**

https://github.com/BloopAI/bloop/assets/43464503/59837381-48dd-4f18-a7e2-1477a5a1b127

**Additional context**
version 0.6.2

**Comment by anastasiya1155** [comment]
Hello @leezhian, thank you for the bug report. We actually now have two types of input, you can choose it in Settings -> Preferences. Can you try using the Simple one?

**Comment by leezhian** [comment]
> 你好@leezhian，感谢您的错误报告。实际上我们现在有两种类型的输入，您可以在“设置”->“首选项”中选择。你能尝试使用简单的吗？

<img width="962" alt="image" src="https://github.com/BloopAI/bloop/assets/43464503/9e1465a0-209d-4996-b308-82ec7d3c9d66">

Is that so?

**Comment by anastasiya1155** [comment]
Yes, this looks correct. I checked again and found the problem. Although it is a simple textarea there is some complicated logic around the persisting chat state across hard rerenders that had a bug for non-Latin input methods. This should be fixed in the next release. Sorry for the inconvenience

**Comment by R3alloc** [comment]
I met the same problem, but the problem only occurred in Simplified Conversation input. When I changed this option into Default, I can use Chinese input method normally. 
![image](https://github.com/BloopAI/bloop/assets/37363760/89f0ed3d-4da8-48af-aa5b-d69bcf19846c)
